### PR TITLE
[fix]:CoreMLModels, No such file or directory

### PR DESCRIPTION
**Pull request by andforce**
fix up CoreMLModels folder error, and remove team id

### CODE REVIEW

File: .gitignore
Comment (mazzzystar) [inline .gitignore:1]: Hi, I think it's better to keep`.gitignore` file in local

File: Queryable/Queryable.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
Comment (mazzzystar) [inline Queryable/Queryable.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: I think it's better to keep files inside `xcshareddata` in local

File: Queryable/Queryable.xcodeproj/project.xcworkspace/xcuserdata/andforce.xcuserdatad/UserInterfaceState.xcuserstate
Comment (mazzzystar) [inline Queryable/Queryable.xcodeproj/project.xcworkspace/xcuserdata/andforce.xcuserdatad/UserInterfaceState.xcuserstate:1]: better to keep files inside xcshareddata in local

File: Queryable/Queryable.xcodeproj/xcuserdata/andforce.xcuserdatad/xcschemes/xcschememanagement.plist
Comment (mazzzystar) [inline Queryable/Queryable.xcodeproj/xcuserdata/andforce.xcuserdatad/xcschemes/xcschememanagement.plist:1]: better to keep files inside xcshareddata in local

**Review by mazzzystar — CHANGES_REQUESTED** [review]
Thanks for your PR, there are some files need to be deleted : )

**Review by mazzzystar — APPROVED** [review]
merged, thx : )

**Comment by andforce** [comment]
> merged, thx : )

效率可真高～

**Pull request merged**